MANIFEST.in

```diff
@@ -1,8 +0,0 @@
-recursive-include readalongs/templates *
-recursive-include readalongs/static *
```

Comment:
do we not need to include readalongs/static? I agree the templates and epub folders aren't here anymore.

Reply:
I don't see the static folder listed in the pyproject file either, but maybe it just get's included automatically or something? has this been tested?

Reply:
Yes, Hatchling doesn't use `MANIFEST.in` from what I can tell: https://hatch.pypa.io/latest/why/#build-backend

So we need to double check to make sure the right things are being included in the source/wheel distributions.

Reply:
Good point, and thanks for the reminder @dhdaines! When Sam and I converted EveryVoice to `pyproject.toml`, we had to do some work to get the wheel and tar ball to contain the right files, and I didn't even test that yet with this PR. Better do that!

I've never understood what MANIFEST.in actually does, I have to admit... 🤷 I suppose I ought to!

Reply:
Checks done: the wheel and tar ball get generated correctly.